### PR TITLE
Backport manual indexes

### PR DESCRIPTION
**Pull request by ericholscher**
These indexes were on the .org, but not the .com.
We hit some performance issues around this,
so I checked to see which indexes we had and backported them into Django to standardize them.

### CODE REVIEW

**Review by stsewd — APPROVED** [review]

File: readthedocs/builds/models.py [hunk 1]
```diff
@@ -693,6 +694,9 @@ class Meta:
             ['version', 'state', 'type'],
             ['date', 'id'],
         ]
+        indexes = [
+            models.Index(fields=['project', 'date']),
+        ]
```
Comment (humitos) [inline readthedocs/builds/models.py:699]: I think this is not required if you are already using `db_index=True` in the fields. https://docs.djangoproject.com/en/3.2/ref/models/options/#indexes

**Review by humitos — APPROVED** [review]
Should we create these indexes manually to do it `CONCURRENTLY` and avoid killing the db?

File: readthedocs/builds/models.py (hunk 1 above)
Comment (ericholscher) [reply readthedocs/builds/models.py:699]: This is indexing them together.

**Comment by ericholscher** [comment]
> Should we create these indexes manually to do it CONCURRENTLY and avoid killing the db?

I don't think these will cause much load -- they are already on .org, and only need to be added to .com.

**Pull request merged**